setup.sh

```diff
@@ -48,6 +48,15 @@ if [[ ! -s "$TFO_MAIN_MODULE_ADDONS/inline-module.tf" ]]; then
                 jq -r --arg key "$key" '.data[$key]' <<< $configmap_json > "${TFO_MAIN_MODULE}/${key}"
             done
         fi
+    # Check if this is a source directory instead of a git repo
+    elif [[ $TFO_MAIN_MODULE_REPO == file://* ]]; then
```

Comment:
In the event of spaces in the var, quote all vars else there is unexpected results. 
- `"$TFO_MAIN_MODULE_REPO"`
- `"${TFO_MAIN_MODULE_REPO#"file://"}"`
- `"$local_module_path"`

Other than this, it looks good.